python/tests/integration/test_methods.py

```diff
@@ -7,18 +7,20 @@
 from PIL import Image  # type: ignore
 import pytest  # type: ignore
 
-from looker_sdk.sdk.api31 import methods as mtds
-from looker_sdk import models as ml
+from looker_sdk.sdk.api31 import models as models31
+from looker_sdk.sdk.api40 import models as models40
 
 
-def test_crud_user(sdk: mtds.LookerSDK):
+def test_crud_user(sdk, models):
```

Comment:
What happens when the models vary here? Is there a union type you can declare?

Reply:
we can declare the type for the `sdk` arg (`sdk: Union[methods31.Looker31SDK, methods40.Looker40SDK]`) but mypy doesn't support using modules as types. I tried `models: Union[methods31, methods40]` and get `Module "looker_sdk.sdk.api31.methods" is not valid as a type` and `Module "looker_sdk.sdk.api40.methods" is not valid as a type`

so I think we'll just have to leave type annotations off the `models` argument here for now. An alternative would be to ditch the `models` fixture and import names directly from either models31/models40 but that would be a lot of extra code and against our practice of importing/using modules rather than names directly from modules. Maybe mypy will implement this at some point?